lib/migration.js

```diff
@@ -893,6 +894,31 @@ function mixinMigration(MySQL, mysql) {
     return columnType;
   }
 
+  function dateOptionsByType(p, columnType) {
+    switch (columnType.toLowerCase()) {
+      default:
+      case 'date':
+        break;
+      case 'time':
+      case 'datetime':
+      case 'timestamp':
```

Comment:
Also should we only add precision fractional second for timestamp values? According to MySQL [documentation](https://dev.mysql.com/doc/refman/5.7/en/datetime.html)
> A DATETIME or TIMESTAMP value can include a trailing fractional seconds part in up to microseconds (6 digits) precision